aopy/data/bmi3d.py

```diff
@@ -548,7 +548,7 @@ def get_target_locations(preproc_dir, subject, te_id, date, target_indices):
             locations[i,:] = trials['target'][trial_idx[0]][[0,2,1]] # use x,y,z format
         else:
             raise ValueError(f"Target index {target_indices[i]} not found")
-    return locations
+    return np.round(locations,2)
```

Comment:
how did you decide on 2 decimal places? i might suggest bumping it up to 4 or 5. whatever will eliminate the 1.00202025e-16 junk but also preserve all the information about where the targets were on the screen.